src/utils/text/index.js

```diff
@@ -62,12 +61,33 @@ export const location2html = (
   if (locations) {
     text += `${locations
       .map(
-        loc =>
+        (loc) =>
           `<p>${loc.fragments
-            .map(f => `${f.start}-${f.end}`)
+            .map((f) => `${f.start}-${f.end}`)
             .join('<br/>')}</p>`,
```

Comment:
Replace `<br/>` by `<br>`? XHTML is so 2010. :p

Reply:
I'll leave it as it is, and we call it JSX?